drivers/hmis/app/graphql/mutations/bulk_merge_clients.rb

```diff
@@ -0,0 +1,30 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Mutations
+  class BulkMergeClients < BaseMutation
+    argument :input, [Types::HmisSchema::ClientMergeInput], required: true
+    field :success, Boolean, null: true
+
+    def resolve(input:)
+      raise 'not allowed' unless current_user.can_merge_clients?
+
+      all_client_ids = input.map(&:client_ids).flatten.uniq
+      clients = Hmis::Hud::Client.viewable_by(current_user).where(id: all_client_ids)
+      raise 'not found' unless clients.size == all_client_ids.size
```

Comment:
I wonder if these checks that are currently size, should be:
```suggestion
      raise 'not found' unless clients.map(:id).uniq.sort == all_client_ids.uniq.sort
```

Reply:
What extra check is that getting us? I don't see a scenario where they would be equal in size but have different values

Reply:
I think you are correct, I just could envision a scenario where there were duplicate IDs in `all_client_ids` and they became unduplicated when pulling the clients.  Belt and suspenders I guess, probably not necessary.